src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -53,6 +53,7 @@
  */
 public class TranscriptServlet extends HttpServlet {
   private DatastoreService datastore;
+  @VisibleForTesting static final String PARAM_ID = "id";
```

Comment:
I think it's better to use `LectureUtil.ID` because the id is based off the redirect Url in `LectureServlet`. If `LectureUtil.ID` changes, `PARAM_ID` would also have to change.

Reply:
Thanks for pointing that out!

This value is now used to generate the queries in javascript, so I think it makes sense to respecify it in each servlet that'll be handling a request with it.

Reply:
Refactored